python/nav/web/devicehistory/utils/componentsearch.py

```diff
@@ -5,15 +5,15 @@
 
 
 def get_component_search_results(
-    search: str, button_suffix: str = '', exclude: list[type[Model]] = []
+    search: str, button_text: str, exclude: list[type[Model]] = []
 ):
     """
     Retrieves grouped search results for component types based on a search query.
 
     Args:
         search (str): The search term to filter components.
-        button_suffix (str, optional): A suffix to append to the button label.
-            Defaults to an empty string.
+        button_text (str): A format string for the button label, which will be formatted
+        with the component type label. It should contain exactly one '%s' placeholder.
```

Comment:
Maybe I didn't see this before now, but please use info field lists compatible with Sphinx for describing parameters in docstrings:
https://www.sphinx-doc.org/en/master/usage/domains/python.html#info-field-lists

Randomly chosen example from NAV:
https://github.com/Uninett/nav/blob/435acb5e6494b7b1ea188a7d6c1233fe17a08fed/python/nav/config.py#L75-L84

Reply:
(this can be used to describe parameter types as well, but I stopped using that once type annotations hit Python)